packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -51,6 +53,48 @@ import {
 } from './utils.js';
 import { OperationVariablesToObject } from './variables-to-object.js';
 
+// converts an ObjectValueNode to the JS object it represents
+const objectNodeToObject = (objectNode: ConstObjectValueNode) => {
+  const { fields } = objectNode;
+  return fields.reduce((acc, field) => {
+    // for some reason this does not seem to match the type
+    const fieldName = field.name as unknown as string;
```

Comment:
Not sure where this is needed? I do see the pattern is already here in some places. Why are some types out of sync?